data/json/mapgen/house06.json

```diff
@@ -87,31 +87,31 @@
-                { "item" : "bed",                  "chance" : 80, "x" : [18,19], "y" : [20,21] },
-                { "item" : "livingroom",           "chance" : 50, "x" : [ 1,10], "y" : [ 8,13] },
-                { "item" : "bedroom",              "chance" : 60, "x" : [ 1, 6], "y" : [15,21] },
-                { "item" : "consumer_electronics", "chance" : 20, "x" : [ 1, 6], "y" : [15,21] },
-                { "item" : "homeguns",             "chance" : 50, "x" : [ 2, 2], "y" : [21,21] },
-                { "item" : "dresser",              "chance" : 50, "x" : [ 2, 2], "y" : [20,20] },
-                { "item" : "dresser",              "chance" : 30, "x" : [ 2, 2], "y" : [17,17] },
-                { "item" : "bedroom",              "chance" : 60, "x" : [15,21], "y" : [17,21] },
-                { "item" : "dresser",              "chance" : 50, "x" : [20,20], "y" : [21,21] },
-                { "item" : "dresser",              "chance" : 50, "x" : [17,17], "y" : [21,21] },
-                { "item" : "cleaning",             "chance" : 70, "x" : [ 7,14], "y" : [17,21] },
-                { "item" : "softdrugs",            "chance" : 50, "x" : [ 7,14], "y" : [17,21] },
-                { "item" : "kitchen",              "chance" : 70, "x" : [15,21], "y" : [ 2,11] },
-                { "item" : "fridge",               "chance" : 70, "x" : [21,21], "y" : [11,11] },
-                { "item" : "oven",                 "chance" : 70, "x" : [21,21], "y" : [10,10] },
-                { "item" : "kitchen",              "chance" : 70, "x" : [21,21], "y" : [ 9, 9] },
-                { "item" : "trash_forest",         "chance" : 70, "x" : [21,21], "y" : [15,15] },
-                { "item" : "homebooks",            "chance" : 80, "x" : [ 2, 5], "y" : [ 6, 6] },
-                { "item" : "office",               "chance" : 80, "x" : [ 4, 6], "y" : [ 3, 3] },
-                { "item" : "cannedfood",           "chance" : 80, "x" : [16,16], "y" : [ 2, 5] },
-                { "item" : "stash_food",           "chance" : 40, "x" : [16,16], "y" : [ 2, 5] },
-                { "item" : "pasta",                "chance" : 40, "x" : [20,20], "y" : [ 2, 5] },
-                { "item" : "stash_food",           "chance" : 40, "x" : [20,20], "y" : [ 2, 5] }
+            "place_loot" : [
+                { "group" : "bed",                  "chance" : 90, "x" : [ 2, 3], "y" : [18,19], "repeat" : [1,3] },
+                { "group" : "bed",                  "chance" : 90, "x" : [18,19], "y" : [20,21], "repeat" : [1,3] },
+                { "group" : "livingroom",           "chance" : 90, "x" : [ 1,10], "y" : [ 8,13], "repeat" : [1,5] },
+                { "group" : "bedroom",              "chance" : 80, "x" : [ 1, 6], "y" : [15,21], "repeat" : [1,4] },
+                { "group" : "consumer_electronics", "chance" : 50, "x" : [ 1, 6], "y" : [15,21], "repeat" : [1,3] },
+                { "group" : "homeguns",             "chance" :  5, "x" : [ 2, 2], "y" : [21,21] },
```

Comment:
`guns_common`. There's plenty of ammo from monsterdrops, gun basements, gun shops etc